docs/apis/plugintypes/mod_data/fields.md

```diff
@@ -1,81 +1,113 @@
 ---
-title: Database fields
-tags:
-  - mod_data
-  - datafield
-  - plugintype
-  - subplugin
-documentationDraft: true
----
```

Comment:
Please do not remove this. This is the page information, tags, title, and other metadata.